web/Services/AdminService.cs

```diff
@@ -89,6 +90,18 @@ public void RejectAdminAccessRequest(int userId)
         userToAdmin.HasRequestedAdmin = false;
         _dbContext.SaveChanges();
     }
+    
+    public void RemoveAdmin(int userId)
+    {
+        var userToUnAdmin = _dbContext.Users.SingleOrDefault(u => u.Id == userId);
+        if (userToUnAdmin == null)
+        {
+            throw new Exception($"No user found with id {userId}");
+        }

```

Comment:
You might be able to replace `SingleOrDefault` with `Single` to avoid throwing your own exception, because `Single` will throw an exception if there isn't exactly one record found.

Reply:
I suspect the exception thrown by using Single gives a less useful message - I was mostly just working from the same logic as the Accept and Reject Admin Access Requests methods, which do it this way.

Reply:
That's true, it's probably fine then! Alternatively, we could show a flash message for it, but this error is unlikely to ever happen so it's probably not worth the effort.